### PR TITLE
feat: Support LockedExternalTask

### PR DESCRIPTION
**Pull request by stefanzilske**
Supports reading variables from LockedExternalTask

### CODE REVIEW

File: extension/core/src/main/java/io/holunda/camunda/bpm/data/CamundaBpmData.java [hunk 1]
```diff
@@ -6,12 +6,7 @@
 import io.holunda.camunda.bpm.data.factory.MapVariableFactory;
 import io.holunda.camunda.bpm.data.factory.SetVariableFactory;
 import io.holunda.camunda.bpm.data.factory.VariableFactory;
-import io.holunda.camunda.bpm.data.reader.CaseServiceVariableReader;
-import io.holunda.camunda.bpm.data.reader.RuntimeServiceVariableReader;
-import io.holunda.camunda.bpm.data.reader.TaskServiceVariableReader;
-import io.holunda.camunda.bpm.data.reader.VariableMapReader;
-import io.holunda.camunda.bpm.data.reader.VariableReader;
-import io.holunda.camunda.bpm.data.reader.VariableScopeReader;
+import io.holunda.camunda.bpm.data.reader.*;
```
Comment (jangalinski) [inline extension/core/src/main/java/io/holunda/camunda/bpm/data/CamundaBpmData.java:9]: Please configure your auto format to not reduce imports to "*".

File: extension/core/src/main/java/io/holunda/camunda/bpm/data/factory/BasicVariableFactory.java [hunk 2]
```diff
@@ -2,15 +2,12 @@
 
 import io.holunda.camunda.bpm.data.adapter.ReadAdapter;
 import io.holunda.camunda.bpm.data.adapter.WriteAdapter;
-import io.holunda.camunda.bpm.data.adapter.basic.ReadWriteAdapterCaseService;
-import io.holunda.camunda.bpm.data.adapter.basic.ReadWriteAdapterRuntimeService;
-import io.holunda.camunda.bpm.data.adapter.basic.ReadWriteAdapterTaskService;
-import io.holunda.camunda.bpm.data.adapter.basic.ReadWriteAdapterVariableMap;
-import io.holunda.camunda.bpm.data.adapter.basic.ReadWriteAdapterVariableScope;
+import io.holunda.camunda.bpm.data.adapter.basic.*;
```
Comment (jangalinski) [inline extension/core/src/main/java/io/holunda/camunda/bpm/data/factory/BasicVariableFactory.java:5]: please do not reduce imports to "*"

File: extension/core/src/main/java/io/holunda/camunda/bpm/data/factory/ListVariableFactory.java [hunk 3]
```diff
@@ -2,15 +2,12 @@
 
 import io.holunda.camunda.bpm.data.adapter.ReadAdapter;
 import io.holunda.camunda.bpm.data.adapter.WriteAdapter;
-import io.holunda.camunda.bpm.data.adapter.list.ListReadWriteAdapterCaseService;
-import io.holunda.camunda.bpm.data.adapter.list.ListReadWriteAdapterRuntimeService;
-import io.holunda.camunda.bpm.data.adapter.list.ListReadWriteAdapterTaskService;
-import io.holunda.camunda.bpm.data.adapter.list.ListReadWriteAdapterVariableMap;
-import io.holunda.camunda.bpm.data.adapter.list.ListReadWriteAdapterVariableScope;
+import io.holunda.camunda.bpm.data.adapter.list.*;
```
Comment (jangalinski) [inline extension/core/src/main/java/io/holunda/camunda/bpm/data/factory/ListVariableFactory.java:5]: please do not reduce imports to "*"

**Review by jangalinski — COMMENTED** [review]
minor issues with import "*" .... lgtm

**Comment by stefanzilske** [comment]
@jangalinski imports fixed.

**Review by zambrovski — APPROVED** [review]

**Pull request merged**